### ISSUE TITLE
Github host key

### ISSUE THREAD

**Issue by madwort** [opensafely/git-server-proxy#2]
I might prefer a signed commit from @sebbacon with a specific host key instead of `StrictHostKeyChecking = accept-new`, what do you think?

https://github.com/opensafely/git-server-proxy/blob/master/setup/make_git_user.sh#L28

**Comment by sebbacon** [comment]
Yes, I think that makes sense. We'd want some kind of tooling to make it clear to the end user that this is what's required, should the host key change, because I'm not sure it'd be obvious to end users otherwise?
